Code/GraphMol/ChemReactions/ReactionRunner.cpp

```diff
@@ -1095,10 +1096,7 @@ void checkAndCorrectChiralityOfMatchingAtomsInProduct(
         // there's a reactant bond that hasn't yet been accounted for:
         int unmatchedBond = -1;
 
-        for (const auto &nbri : boost::make_iterator_range(
-                 reactant.getAtomNeighbors(&reactantAtom))) {
-          const Bond *rBond =
-              reactant.getBondBetweenAtoms(reactantAtom.getIdx(), nbri);
+        for (const auto &rBond : reactant.atomBonds(&reactantAtom)) {
```

Comment:
These aren't references, are they? (that's a warning-to-be)

Reply:
correct, will fix that